### PR TITLE
Unit selection

### PR DESCRIPTION
**Pull request by tmjo**
Adding unit selection to config flow and let user select between kW/W and kWh/Wh. Moved some variables to const too.

### CODE REVIEW

**Comment by fondberg** [comment]
@tmjo could be good to start using branches directly with thia repo. Much easier then a fork

File: custom_components/easee/sensor.py [hunk 1]
```diff
@@ -53,11 +60,18 @@ async def async_setup_entry(hass, entry, async_add_entities):
                 )
 
         monitored_days = config.options.get(MEASURED_CONSUMPTION_DAYS, [])
+        consumption_unit = "kWh"
```
Comment (fondberg) [inline custom_components/easee/sensor.py:63]: These 3 lines can be written as one line

**Review by fondberg — APPROVED** [review]

File: custom_components/easee/sensor.py (hunk 1 above)
Comment (tmjo) [reply custom_components/easee/sensor.py:63]: like

`consumption_unit = CUSTOM_UNITS_TABLE[consumption_unit] if consumption_unit in custom_units else "kWh"`

or something smarter?

**Comment by tmjo** [comment]
> @tmjo could be good to start using branches directly with thia repo. Much easier then a fork

Puuuh :) To much new at the same time, but I'll try on the next one.

**Comment by fondberg** [comment]
Hehe

Of course the next one. Benefit is that all branches are in one repo which makes it easier tbe next time for you

File: custom_components/easee/sensor.py (hunk 1 above)
Comment (fondberg) [reply custom_components/easee/sensor.py:63]: I think so.

**Pull request merged**